### PR TITLE
Intersection of two Mobjects

### LINKED ISSUE #27: intersection points
ֆունկցիա, որը 2 VMobject-ների հատման կետի կոորդինատները տա, ասենք ուղղի ու շրջանագծի հատման կետ, շրջանագծերի հետման կետեր և այլն

### PR DESCRIPTION
**Pull request by zidder**
Closes #27

### CODE REVIEW

**Comment by zidder** [comment]
Demo: 
https://user-images.githubusercontent.com/28303059/188119451-4d335337-562a-4b48-99c2-f3d6fb99ba82.mp4

**Review by a-arakelian — APPROVED** [review]
I didn't fully understand the content. But, as no one comments, I make it possible to merge, assuming that everything is in working order

**Comment by zidder** [comment]
@a-arakelian 
Let me summarise:
- I moved scenes to a folder called `scenes/`
- I created another folder besides `enumerated/`, now we can create others as well
- I made it possible to run scenes with path instead of problem number
- I've created a Scene, which demonstrates the result of the algorithm, finding intersections of two objects. I've picked several objects and intersected them. The video is in the previous comment.

If you have any suggestions You think I should try, go ahead and tell me. Otherwise I can move the function to the `lib/` folder, and merge it. Will wait for some reply before merging

**Pull request merged**
